salt/metalk8s/internal/sreport/files/metalk8s.py

```diff
@@ -0,0 +1,123 @@
+from sos.plugins import Plugin, RedHatPlugin
+from os import path
+
+
+class metalk8s(Plugin, RedHatPlugin):
+
+    """Metalk8s plugin
+    """
+
+    plugin_name = 'metalk8s'
+    packages = ('kubernetes', 'kubernetes-master')
+    profiles = ('container',)
+    files = ("/etc/kubernetes/admin.conf",)
+
+    option_list = [
+        ("all", "also collect all namespaces output separately",
+            'slow', False),
+        ("describe", "capture descriptions of all kube resources",
+            'fast', False),
+        ("podlogs", "capture logs for pods", 'slow', False),
+    ]
+
+    def check_is_master(self):
+        if any([
+            path.exists("/etc/kubernetes/admin.conf")
+        ]):
+            return True
+        return False
+
+    def setup(self):
+        self.add_copy_spec("/etc/kubernetes")
+        self.add_copy_spec("/var/log/pods")
+        self.add_copy_spec("/var/log/metalk8s-bootstrap.log")
+
+        svcs = [
```

Comment:
svcs? do you mean `services` (if yes, then use `services`, it's way more understandable)